.github/workflows/clean.yml

```diff
@@ -7,9 +7,9 @@ jobs:
   clean:
     permissions:
       contents: read
-    uses: hpi-schul-cloud/dof_app_deploy/.github/workflows/clean_workflow.yml@main
+    uses: hpi-schul-cloud/dof_app_deploy/.github/workflows/clean_workflow.yml@BC-4710-new-tldraw-manage
```

Comment:
This will need to be changed to main before merging, right?

Reply:
yes